reservations.htm

```diff
@@ -24,6 +24,26 @@
 
+<div align=center style="margin:20px auto">
+<input type=hidden name=respage id=respage value="https://v2.reservationkey.com/sherwoodcourtinn/reserve">
+<script src="http://v2.reservationkey.com/inc/dpicker.js"></script> 
+<script src="http://v2.reservationkey.com/inc/jsdate.js"></script> 
+<link REL="stylesheet" HREF="http://v2.reservationkey.com/inc/dpicker.css" TYPE="text/css" /> 
+<table cellpadding=0 cellspacing=0 border=0 align=center><tr>
+<td bgcolor=bbbbbb width=3 height=3><img src="http://v2.reservationkey.com/i/admintlc_bl.gif"></td><td bgcolor=bbbbbb height=3></td> 
+<td bgcolor=bbbbbb width=3 height=3><img src="http://v2.reservationkey.com/i/admintrc_bl.gif"></td></tr><tr>
+<td bgcolor=bbbbbb width=3></td> <td bgcolor=bbbbbb>
+<table cellpadding=0 cellspacing=0 border=0 align=center style="border-right: white 1px solid; border-top: white 1px solid; border-left: white 1px solid; border-bottom: white 1px solid; width: 510px; style="border-right: black 1px solid; border-top: black 1px solid; border-left: black 1px solid; border-bottom: black 1px solid; width: 277px; background-color: ;""> <tr><td align=center nowrap> 
+Show availability during the following date range:<br><div class=text style="font-weight:bold"> 
+Arrival Date: <input type=text name="checkin" id="checkin" value="" size=7 maxlength=20 onclick="displayDatePicker('checkin', false, 'mdy', '/');"> <a href="javascript:void(0)" onclick="displayDatePicker('checkin', false, 'mdy', '/');"><img src="http://v2.reservationkey.com/i/cal.gif" border=0></a>   
+Departure Date: <input type=text name="checkout" id="checkout" value="" size=7 maxlength=20 onclick="displayDatePicker('checkout', false, 'mdy', '/');"> <a href="javascript:void(0)" onclick="displayDatePicker('checkout', false, 'mdy', '/');"><img src="http://v2.reservationkey.com/i/cal.gif" border=0></a>   
+Guests: <select name="numguests" id="numguests"><option value="1">1</option><option value="2" selected>2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option><option value="6">6</option><option value="7">7</option><option value="8">8</option><option value="9">9</option><option value="10">10</option></select>
+<input type=button value="Search" onclick="window.location=document.getElementById('respage').value +'/s|'+ document.getElementById('checkin').value.replace(/\//g,'.')+'|'+ document.getElementById('checkout').value.replace(/\//g,'.')+'|'+ document.getElementById('numguests').value">
+</td></tr></table></td><td bgcolor=bbbbbb width=3></td></tr><tr><td bgcolor=bbbbbb width=3 height=3><img src="http://v2.reservationkey.com/i/adminblc_bl.gif"></td>
+<td bgcolor=bbbbbb height=3></td><td bgcolor=bbbbbb width=3 height=3><img src="http://v2.reservationkey.com/i/adminbrc_bl.gif"></td></tr></table></td></tr></table>
```

Comment:
There was an extra `</td></tr></table>` at the end of this line, which is closing the containing `<table>` too ...

![screen shot 2016-01-22 at 8 56 58 pm](https://cloud.githubusercontent.com/assets/71928/12527876/b7411360-c14a-11e5-84ea-a6ac1d6d6c83.png)

I'll add another commit to remove those and it should look fine.